init.lua

```diff
@@ -80,6 +160,7 @@ for _, v in pairs({
 		color = "#8afb94ff",
 		tiles = {tile},
 		place_param2 = 136,
+		groups = {snappy = 3, flammable = 2, leaves = 1, leafdecay = 1},
```

Comment:
this shouldn't be needed. We don't want to override unrelated properties of the nodes.

Reply:
Of course preserving the normal behaviors of leafdecay is really important.   Said with a little bit of sarcasm.